api/compute/v1alpha1/backendconfig_types.go

```diff
@@ -33,6 +33,9 @@ type BackendConfigSpec struct {
 
 	// +kubebuilder:validation:Optional
 	Env map[string]string `json:"env,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	Liveness *Liveness `json:"liveness,omitempty"`
```

Comment:
how about we move them into Pod field so it could get consistent as the function and connector CRDs

Reply:
updated